crates/sui-benchmark/tests/simtest.rs

```diff
@@ -4,6 +4,7 @@
 #[cfg(msim)]
 mod test {
 
```

Comment:
simtest found an issue in [fullnode test](https://github.com/MystenLabs/sui/blob/main/crates/sui/tests/full_node_tests.rs#L1204)
In current live pruner implementation we remove old versions ASAP.
This test tries to read old version of the object from the full node and expectedly fails.
[Comment](https://github.com/MystenLabs/sui/blob/43d8ef5c8c6bc453b13c45b373166801b959c336/crates/sui-core/src/authority.rs#L1929) around the impl function says that it's the best effort function anyway.
So what's a suggestion here? Should we just deprecate the function and corresponding json rpc api or disable live pruning on full nodes for now? 

cc @mlogan, @sadhansood 

*starting thread here cuz github doesn't support commenting on files not in PR*

Reply:
we could also temporary disable this test
if we land PR as it is, production behaviour won't change because we're using `ignore_range_deletions` setting
but we need to decide what to do with this API before switching pruner from delete_range to point-delete

Reply:
I'd recommend disabling pruning for that test. There is a config builder which we should be able to use.

Reply:
sure, we can add extra methods to TestClusterBuilder and functionality on top to disable live pruner, but I don't think it's worth it. 
And original question remains, what do we do with corresponding json rpc api going forward?